scripts/ci

```diff
@@ -4,6 +4,12 @@ set -e
 source $(dirname $0)/lib/debug_functions
 
 cd $(dirname $0)
+
+if [[ $5 = operator ]]; then
+   test -d ../operators/go/submariner-operator || ./codegen-operator
```

Comment:
If the assumption here is that the code generation is capable of producing the operator, I think we should always re-generate it during CI. (Once we switch to a static operator, this should be removed, or perhaps used to check the differences...)

Reply:
Done in the latest